src/components/Router.js

```diff
@@ -25,7 +25,8 @@ import * as NotebookLauncher from 'src/pages/workspaces/workspace/notebooks/Note
 import * as TerminalLauncher from 'src/pages/workspaces/workspace/notebooks/TerminalLauncher'
 import * as Tools from 'src/pages/workspaces/workspace/Tools'
 import * as WorkflowView from 'src/pages/workspaces/workspace/tools/WorkflowView'
-import * as Billing from 'src/pages/BillingProjectsList'
+import * as BillingProjectsList from 'src/pages/Billing/projectsList'
```

Comment:
We should match the file naming from Groups, with files called `List` and `Project`